src/vs/workbench/browser/layout.ts

```diff
@@ -646,6 +665,7 @@ export class WorkbenchLayout extends Disposable implements IVerticalSashLayoutPr
 
 		// Propagate to Part Layouts
 		this.parts.titlebar.layout(new Dimension(this.workbenchSize.width, this.titlebarHeight));
+		this.parts.menubar.layout(new Dimension(undefined, this.menubarHeight));
```

Comment:
@sbatten I would put the width here too if it applies (possibly the width of the workbench?)